JetStreamCompose/gradle/libs.versions.toml

```
@@ -1,40 +1,34 @@
 [versions]
-activity-compose = "1.9.0"
-android-gradle-plugin = "8.4.1"
-android-test-plugin = "8.4.1"
-androidx-baselineprofile = "1.2.4"
-benchmark-macro-junit4 = "1.2.4"
+activity-compose = "1.9.2"
+android-gradle-plugin = "8.6.0"
```

nit, but you use test plugin newer version below this line :)

```suggestion
android-gradle-plugin = "8.6.1"
```